src/App.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans&display=swap');
+
+*{
+    margin: 5px;
+    padding: 0;
+}
+body{
+    background-color: #ffc74a;
+}
+header {
+    height: auto;
```

Comment:
Als je de width en height op `auto` zet hoef je het niet te benoemen, want dat is de `default` :)